Doesnt build on OpenBSD

Wanted to try exa on OpenBSD, but the users crate obviously fails to build:

```
[14:58] c64:/tmp/rust-users/ $cargo build                          
    Updating registry `https://github.com/rust-lang/crates.io-index`
   Compiling libc v0.2.21                                                                                                                     Compiling users v0.5.2 (file:///tmp/rust-users)              
error[E0412]: type name `c_char` is undefined or not in scope
  --> src/base.rs:75:23                                              
   |                                                       
75 |     gr_name:   *const c_char,         // group name
   |                       ^^^^^^ undefined or not in scope
...

error[E0412]: type name `c_passwd` is undefined or not in scope
  --> src/base.rs:82:39
   |
82 |     fn getpwuid(uid: uid_t) -> *const c_passwd;
   |                                       ^^^^^^^^ undefined or not in scope
...
error[E0412]: type name `os::UserExtras` is undefined or not in scope
   --> src/base.rs:105:13                                  
    |                                                   
105 |     extras: os::UserExtras,                          
    |             ^^^^^^^^^^^^^^ undefined or not in scope    
...
error[E0412]: type name `os::GroupExtras` is undefined or not in scope
   --> src/base.rs:165:13  
    |                              
165 |     extras: os::GroupExtras,
    |             ^^^^^^^^^^^^^^^ undefined or not in scope  

error[E0433]: failed to resolve. Could not find `UserExtras` in `os`
   --> src/base.rs:247:28
    |                                                             
247 |             extras:        os::UserExtras::from_passwd(passwd),                      
    |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Could not find `UserExtras` in `os`

```
This is with rustc 1.15.1. Yay manual os whitelisting in base.rs...... :/

--- a/src/base.rs
+++ b/src/base.rs
@@ -36,14 +36,14 @@ use std::sync::Arc;
 
 use libc::{uid_t, gid_t};
 
-#[cfg(any(target_os = "macos", target_os = "freebsd", target_os = "dragonfly"))]
+#[cfg(any(target_os = "macos", target_os = "freebsd", target_os = "dragonfly", target_os = "openbsd"))]
 use libc::{c_char, time_t};
 
 #[cfg(target_os = "linux")]
 use libc::c_char;
 
 
-#[cfg(any(target_os = "macos", target_os = "freebsd", target_os = "dragonfly"))]
+#[cfg(any(target_os = "macos", target_os = "freebsd", target_os = "dragonfly", target_os = "openbsd"))]
 #[repr(C)]
 pub struct c_passwd {
     pw_name:    *const c_char,  // user name
@@ -461,7 +461,7 @@ pub mod os {
     /// Although the `passwd` struct is common among Unix systems, its actual
     /// format can vary. See the definitions in the `base` module to check which
     /// fields are actually present.
-    #[cfg(any(target_os = "linux", target_os = "macos", target_os = "freebsd", target_os = "dragonfly"))]
+    #[cfg(any(target_os = "linux", target_os = "macos", target_os = "freebsd", target_os = "dragonfly", target_os = "openbsd"))]
     pub mod unix {
         use std::path::Path;
 
@@ -595,7 +595,7 @@ pub mod os {
     ///
     /// These platforms have `change` and `expire` fields in their `passwd`
     /// C structs.
-    #[cfg(any(target_os = "macos", target_os = "freebsd", target_os = "dragonfly"))]
+    #[cfg(any(target_os = "macos", target_os = "freebsd", target_os = "dragonfly", target_os = "openbsd"))]
     pub mod bsd {
         use std::path::Path;
         use libc::time_t;
@@ -680,7 +680,7 @@ pub mod os {
     }
 
     /// Any extra fields on a `User` specific to the current platform.
-    #[cfg(any(target_os = "macos", target_os = "freebsd", target_os = "dragonfly"))]
+    #[cfg(any(target_os = "macos", target_os = "freebsd", target_os = "dragonfly", target_os = "openbsd"))]
     pub type UserExtras = bsd::UserExtras;
 
     /// Any extra fields on a `User` specific to the current platform.
@@ -688,7 +688,7 @@ pub mod os {
     pub type UserExtras = unix::UserExtras;
 
     /// Any extra fields on a `Group` specific to the current platform.
-    #[cfg(any(target_os = "linux", target_os = "macos", target_os = "freebsd", target_os = "dragonfly"))]
+    #[cfg(any(target_os = "linux", target_os = "macos", target_os = "freebsd", target_os = "dragonfly", target_os = "openbsd"))]
     pub type GroupExtras = unix::GroupExtras;
 }
 

